### PR TITLE
write test

### PR DESCRIPTION
**Pull request by AlinkaLevchenko**

### CODE REVIEW

File: src/test/java/AndriiTest/DeliveryDetailsPage.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public DeliveryDetailsPage(WebDriver driver) {
     private static final String TERMS_AND_CONDITIONS_CHECKBOX = "//input[contains(@id, 'TermsAndConditions')]";
     private static final String PROCEED_TO_PAYMENT_BTN = "//a[@title='Proceed to Payment']";
 
-    // Test User
+    // LevchenkoAlinka.Test User
```
Comment (Wolmeright) [inline src/test/java/AndriiTest/DeliveryDetailsPage.java:25]: Delete "LevchenkoAlinka"

File: src/test/java/LevchenkoAlinka/Pages/HomePageInteriorDesign.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class HomePageInteriorDesign extends BasePage {
+    public HomePageInteriorDesign(WebDriver driver) {
```
Comment (Wolmeright) [inline src/test/java/LevchenkoAlinka/Pages/HomePageInteriorDesign.java:11]: Use constructor from BasePage

File: src/test/java/LevchenkoAlinka/Tests/TestInteriorDesign.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import BasePageAndTestInit.TestInit;
+import LevchenkoAlinka.Pages.HomePageInteriorDesign;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestInteriorDesign extends TestInit {
+
+    @Test
+    public void designApartmentInSmartRoom() {
+        HomePageInteriorDesign homePageInteriorDesign = new HomePageInteriorDesign(driver);
```
Comment (Wolmeright) [inline src/test/java/LevchenkoAlinka/Tests/TestInteriorDesign.java:11]: Add lines before and after 11 line

File: src/test/java/LevchenkoAlinka/Tests/TestInteriorDesign.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        openUrl("https://polyakova.biz/ua");
+        homePageInteriorDesign.portfolioBtn().click();
+        homePageInteriorDesign.smartRoomsopen().click();
+        homePageInteriorDesign.roomInCampus().click();
+
+        Assert.assertTrue(homePageInteriorDesign.photoDesignFlatCampus().isDisplayed());
+    }
+}
+
+
```
Comment (Wolmeright) [inline src/test/java/LevchenkoAlinka/Tests/TestInteriorDesign.java:21]: Delete empty lines

**Comment by AlinkaLevchenko** [comment]
error

**Pull request closed without merging**